apps/prairielearn/assets/scripts/lib/editAssessmentQuestionsTable.ts

```diff
@@ -0,0 +1,374 @@
+import { html } from '@prairielearn/html';
+
+import { TagBadgeList } from '../../../src/components/TagBadge.html.js';
+import { TopicBadge } from '../../../src/components/TopicBadge.html.js';
+
+function maxPoints({
+  max_auto_points,
+  max_manual_points,
+  points_list,
+  init_points,
+  assessmentType,
+}: {
+  max_auto_points: number;
+  max_manual_points: number;
+  points_list: number[];
+  init_points: number;
+  assessmentType: string;
+}) {
+  if (max_auto_points || !max_manual_points) {
+    if (assessmentType === 'Exam') {
+      return (points_list || [max_manual_points]).map((p) => p - max_manual_points).join(',');
+    }
+    if (assessmentType === 'Homework') {
+      return `${init_points - max_manual_points}/${max_auto_points}`;
+    }
+  } else {
+    return html`&mdash;`;
+  }
+}
+
+export function EditAssessmentQuestionsTable({
+  zones,
+  assessmentType,
+  showAdvanceScorePercCol,
+}: {
+  zones: any[];
```

Comment:
Same comment as the client file: better types please!

Reply:
Updated types in [f6c763c](https://github.com/PrairieLearn/PrairieLearn/pull/10255/commits/f6c763c5f992e9f7881c25f1ae320ca10d7a0e99) and [8f0f1ad](https://github.com/PrairieLearn/PrairieLearn/pull/10255/commits/8f0f1addcf68df484e13a9397328a086221a8248).